frontend/src/components/Login.tsx

```diff
@@ -51,61 +55,109 @@ const Login: React.FC<Props> = () => {
+        <div className="col-md-5">
+          <div className="col-md-10 mx-auto">
+            <Formik
+              initialValues={initialValues}
+              validationSchema={validationSchema}
+              onSubmit={handleLogin}
+            >
+              {({ handleSubmit }) => (
+              <Form onSubmit={handleSubmit}>
+              <div className="form-floating mb-3">
+                <Field
+                  name="username"
+                  type="text"
+                  className="form-control input-lg"
+                  id="floatingUsername"
+                  placeholder=" "
+                  style={{ borderRadius: "15px" }}
+                />
+                <label
+                  htmlFor="floatingUsername"
+                  style={{
+                    paddingLeft: '10px',
+                    paddingTop: 0,
+                    paddingBottom: 0,
+                    paddingRight: 0,
+                    lineHeight: '3rem',
+                    margin: 0,
+                  }}
+                >
+                  Username
+                </label>
+              </div>
+
+              <div className="form-floating">
+                <Field
+                  name="password"
+                  type="password"
+                  className="form-control input-lg"
+                  id="floatingPassword"
+                  placeholder=" "
+                  style={{ borderRadius: "15px" }}
+                />
+                <label
+                  htmlFor="floatingPassword"
+                  style={{
+                    paddingLeft: '10px',
+                    paddingTop: 0,
+                    paddingBottom: 0,
+                    paddingRight: 0,
+                    margin: 0,
+                    lineHeight: '3rem',
+                  }}
+                >
+                  Password
+                </label>
+              </div>
+              
+
+              <div className="form-group mt-3 d-flex align-items-center">
+              <div className="col-md-5" style={{ fontFamily: 'Inter, sans-serif', fontWeight: 'bold', color: 'rgba(0, 0, 0, 0.5)'  }}>
+                No Account?
+                <br/>
+                <Link to="/register" style={{ textDecoration: 'underline', color: 'rgba(0, 0, 0, 0.5)' }}>
+                  Sign up
+                </Link>
+              </div>
+                <div className="col-md-2" />
+                <div className="col-md-5 text-center">
+                  <button
+                    type="submit"
+                    className="btn btn-block rounded-pill"
+                    style={{
+                      backgroundColor: '#6C63FF',
+                      color: 'white',
+                      fontFamily: 'Inter, sans-serif',
+                      fontWeight: 'bold',
+                      height: '50px',
+                    }}
+                  >
+                    Login
+                  </button>
                 </div>
```

Comment:
Should we handle the displaying of login errors here also?

(E.G. if the username does not exist)